person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(id, age, name = 'Unknown', parent_permission: true)
+    @id = id
+    @age = age
+    @name = name
+    @parent_permission = parent_permission
+  end
+
+  def can_use_service?
+    of_age? || @parent_permission
+  end
+
+  private
+
+  def of_age?
+    age >= 18
+  end
+
+  attr_reader :id
+  attr_writer :name, :age
```

Comment:
- [ ] As per these [requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md#:~:text=and%20true.-,Getters%20for%20%40id%2C%20%40name%2C%20and%20%40age.,Setters%20for%20%40name%20and%20%40age.,-Private%20method%20of_age), `name` and `age` instance variables should have getters and setters attribute `attr_accessor`, instead of setters only `attr_writer` as you did. Please fix that. 
- [ ] [Optional] [Highly recommended] It would be better professionally and considered good practice to move the getters and setters attributes to the top of the class even before the initialize method. it will be like that:

```ruby
class Person
  attr_accessor :name, :age 
  attr_reader :id 
  
  ------------------
  ------------------
end
```